HelpMyStreetFE/HelpMyStreetFE/src/js/profile/requests.js

```diff
@@ -40,4 +42,35 @@ export function initialiseRequests() {
       $(`#job-${id} .job__detail`).slideToggle();
     });
   });
+
+
+    $('.accept-request').click(function (evt) {
+        if ($(this).attr("data-accepted") != "true") {
```

Comment:
would it be better to add `disabled` to the button so it can't be clicked at all? Or even hide the job all together with a `slideUp()` or remove the html element altogether?

Reply:
i've changed it to disabled - the requirements just want the button to say "accepted" until they navigate away from the page.